### DISCUSSION TITLE
Could not resolve net.minestom:minestom-snapshots:f1d5940855

### DISCUSSION THREAD

**Discussion by Obscrts** [Minestom/Minestom#2206] [Help] [answered]
Hey there,
I am trying to setup a base project of Minestom and my build.gradle.kts looks like this:
`plugins {
    id("java")
}

group = "net.astraleth.studios"
version = "0.1-SNAPSHOT"

repositories {
    mavenCentral()
    maven(url = "https://jitpack.io")
}

dependencies {
    implementation("net.minestom:minestom-snapshots:f1d5940855")
}`

but I'm getting this error:
`Astralis:main: Could not resolve net.minestom:minestom-snapshots:f1d5940855.
Required by:
    project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html`
 
 Am I doing something wrong?

**Comment by Marfien** [comment, marked as answer]
Hey @Obscrts, looks like you need to declare your java version fist. Smth. like this:
```kotlin
java {
    toolchain {
        languageVersion = JavaLanguageVersion.of(21)
    }
}
```
